### PR TITLE
Add citation for Paper 3

### PR DESCRIPTION
**Pull request by adrn**
This adds the bibtex for Paper 3 to the Acknowledging and Citing Astropy page.

### CODE REVIEW

File: acknowledging.html [hunk 1]
```diff
@@ -170,28 +190,32 @@ <h1 id="acknowledging-or-citing-astropy">Acknowledging or Citing Astropy<a class
...
+                <li><a href="https://arxiv.org/abs/1801.02634" target="_blank">Astropy Paper II (v2.0)</a>
                     (<a href="http://adsabs.harvard.edu/abs/2018arXiv180102634T" target="_blank">ADS</a> -
-                    <a href="http://adsabs.harvard.edu/cgi-bin/nph-bib_query?bibcode=2018arXiv180102634T&data_type=BIBTEX&db_key=PRE&nocookieset=1" target="_blank">BibTeX</a>)
+                    <a href="https://ui.adsabs.harvard.edu/abs/2018AJ....156..123A/exportcitation" target="_blank">BibTeX</a>)
                 </li>
-                <li><a href="https://doi.org/10.1051/0004-6361/201322068" target="_blank">Astropy Paper I</a>
+                <li><a href="https://doi.org/10.1051/0004-6361/201322068" target="_blank">Astropy Paper I (v0.2)</a>
                     (<a href="http://adsabs.harvard.edu/abs/2013A%26A...558A..33A" target="_blank">ADS</a> -
-                    <a href="http://adsabs.harvard.edu/cgi-bin/nph-bib_query?bibcode=2013A%26A...558A..33A&amp;data_type=BIBTEX&amp;db_key=AST&amp;nocookieset=1" target="_blank">BibTeX</a>)
+                    <a href="http://adsabs.harvard.edu/abs/2013A%26A...558A..33A/exportcitation" target="_blank">BibTeX</a>)
```
Comment (pllim) [inline acknowledging.html:205]: Why not make all the URLs consistent?
```suggestion
                    <a href="https://ui.adsabs.harvard.edu/abs/2013A%26A...558A..33A/exportcitation" target="_blank">BibTeX</a>)
```

Comment (adrn) [reply acknowledging.html:208]: Sure - I think this was just leftover from the old ADS interface

**Comment by adrn** [comment]
See also: astropy/astropy#13588

**Review by pllim — APPROVED** [review]
LGTM. Thanks!

I won't click merge in case you want more people to review.

**Comment by adrn** [comment]
Thanks for looking! I'm not sure who else should review - maybe @eteq?

**Review by dhomeier — APPROVED** [review]

**Review by xuanxu — APPROVED** [review]
👍

**Comment by adrn** [comment]
Two approvals, and I confirmed locally that it renders ok and the copy to bibtex button works - merging!

**Pull request merged**